Header header

bool colliding
string[] collidingLinks
